README.md

```diff
@@ -1,25 +1,52 @@
 # xmlvirshparser
 
-Python script to parse the output of "virsh dumpxml" command captured with sosreport
+Python script to parse the output of `virsh dumpxml` command captured with sosreport
 
-# Install
```

Comment:
Ah! looks like steps a repeated.

What I'm looking for is

# Installation
## Using `pip`
Activate python virtual environment and type
```
pip install xmlvirshparser
```
## Manual install
Clone this repository by typing
```
git clone https://github.com/ervikrant06/virsh_xml_parser.git
```

Create Python virtual environment
```
# for Python 2.7
virtualenv ~/myvirtualenvs/xmlvirshparser

# for Python 3.5
pyvenv ~/myvirtualenvs/xmlvirshparser
```

Activate Python virtual environment
```
source ~/myvirtualenvs/xmlvirshparser/bin/activate
```

Install package using
```
cd virsh_xml_parser
python setup.py install
```